### ISSUE TITLE
EventTarget error when using modules under Node.js

### ISSUE THREAD

**Issue by djipco** [ShiftLimits/wled-client#4]
I'm trying to use **wled-client** in Node.js with `type: module` defined in my `package.json`. When I run my code, I get this error: 
```
TypeError: ee.default is not a constructor
    at EventTarget.connect (file:///Users/jpcote/CODE/node_modules/wled-client/dist/browser/wled-client.mjs:1:7496)
    at EventTarget.init (file:///Users/jpcote/CODE/node_modules/wled-client/dist/browser/wled-client.mjs:1:20808)
    at new We (file:///Users/jpcote/CODE/node_modules/wled-client/dist/browser/wled-client.mjs:1:20672)
    at new Tubes (file:///Users/jpcote/CODE/src/Tubes.js:9:17)
    at file:///Users/jpcote/CODE/index.js:5:15
    at ModuleJob.run (node:internal/modules/esm/module_job:198:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:385:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)

Node.js v17.8.0
```

As you can see, it tries to use the browser version instead of the Node.js version. If I modify the `package.json` file of **wled-client** to make the Node version the default, it works fine:

```json
  "exports": {
    "node": {
      "module": "./dist/node/wled-client.js",
      "require": "./dist/node/wled-client.cjs"
    },
    "script": "./dist/browser/wled-client.js",
    "default": "./dist/node/wled-client.js"  <---------- THIS WORKS!
  },
```
I guess something isn't right with the way it detects which version to use. 

P.S. Thanks for this cool library.

**Comment by ShiftLimits** [comment]
Hello, thanks for reporting this!

Node is supposed to be using the "import" entry point defined under the "exports" property of package.json. It looks like I mistakenly used the word "module" instead of "import", which lead to Node ending up at the "default" entry point which is meant for browsers.

This should be fixed now in 0.22.1 which should address your issue.

**Comment by djipco** [comment]
Yep! It did fix the issue. Thank you for the quick turnaround.
